core/src/main/java/org/mule/endpoint/AbstractEndpointBuilder.java

```diff
@@ -297,22 +301,49 @@ private String getDynamicUriFrom(String uri)
     }
 
     
-    protected List<MessageProcessor> addTransformerProcessors(EndpointURI endpointURI)
-        throws TransportFactoryException
+    protected List<MessageProcessor> addTransformerProcessors(EndpointURI endpointURI) throws TransportFactoryException
     {
         List<MessageProcessor> tempProcessors = new LinkedList<MessageProcessor>(messageProcessors);
         tempProcessors.addAll(getTransformersFromUri(endpointURI));
         tempProcessors.addAll(transformers);
+
+        for (MessageProcessor mp : tempProcessors)
```

Comment:
Do this means that objects that where not registered in the registry before are now being registered?